package.json

```
@@ -105,7 +109,7 @@
     "eslint-plugin-react-native": "^4.1.0",
     "expo": "~51.0.17",
     "react-native-svg-transformer": "^1.5.0",
-    "typescript": "^5.5.4"
+    "typescript": "^5.3.3"
```

Peut-être mettre à jour vers la dernière version pour résoudre tous les bugs ?
```suggestion
    "typescript": "^5.6.3"
```

---

ne change rien 😞...